bpf/lib/common.h

```diff
@@ -672,6 +676,20 @@ enum metric_dir {
 #define DSR_IPV6_OPT_LEN	(sizeof(struct dsr_opt_v6) - 4)
 #define DSR_IPV6_EXT_LEN	((sizeof(struct dsr_opt_v6) - 8) / 8)
 
+/* Geneve option used to carry service addr and port for DSR.
+ *
+ * Class = 0x014B (Cilium according to [1])
+ * Type  = 0x1   (vendor-specific)
+ *
+ * [1]: https://www.iana.org/assignments/nvo3/nvo3.xhtml#geneve-option-class
+ */
+#define DSR_GENEVE_OPT_CLASS	0x014B
+#define DSR_GENEVE_OPT_TYPE	0x81
```

Comment:
In case you need to push one last time, maybe make this

```
#define GENEVE_OPT_TYPE_CRIT 0x80
...
#define DSR_GENEVE_OPT_TYPE (GENEVE_OPT_TYPE_CRIT | 0x01)
```

Reply:
Done